### ISSUE TITLE
Does this plugin still work with VS Code 1.6?

### ISSUE THREAD

**Issue by djmtype** [mattbierner/vscode-color-info#31]
When hovering over any color value in a .css file, no preview gets returned. Is there some other setting that needs to be enabled in VS Code? 

Using –

Version: 1.65.2
Commit: c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1
Date: 2022-03-10T14:42:05.236Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Darwin arm64 21.4.0

**Comment by mattbierner** [comment]
The color info hovers works fine for me:

<img width="693" alt="Screenshot 2023-01-17 at 11 17 13 PM" src="https://user-images.githubusercontent.com/212305/213107801-936a9c4c-19d8-4e48-bdef-a3fd5ba0653f.png">

Try checking your settings to make sure it is enabled in css files

**Comment by djmtype** [comment]
@mattbierner That's very strange. I removed all my settings from VS Code with the exception of your plugin. Still not receiving the dialog. I tried removing VS Code's color box preview too in case that was conflicting. `"editor.colorDecorators": false`

<img width="1600" alt="Screenshot 2023-01-18 at 8 55 27 AM" src="https://user-images.githubusercontent.com/930713/213190772-f08ceef8-086e-44e1-82e8-8d81217045ae.png">
